README.md

```diff
@@ -459,40 +528,9 @@ HTTP/1.1 302 Found
   &redirect_uri=https%3A%2F%2Fclient.example.org%2Fcb
 ```
 
-#### Authentication Response (`vp_ldp`)
-
-ID Token would be same as the example in the Self-Issued OP with Verifiable Presentation in ID Token, Authentication Response (`vp_jwt`) section.
-
-
-#### Claims parameter (vp_ldp)
-
-Below is a non-normative example of how the `claims` parameter can be used for requesting verified presentations signed as Linked Data Proofs.
-
-```json
-{
-   "id_token":
-    {
-      "vp_ldp": {
-        "credential_types": ["https://did.itsourweb.org:3000/smartcredential/Ontario-Health-Insurance-Plan"]
-        "claims":
-        {
-          "given_name": null,
-          "family_name": null,
-          "birthdate": null
-        }
-      }
-    }
-}
-```
-
-#### Authentication Response (`vp_ldp`)
-
-ID Token would be same as the example in the Self-Issued OP with Verifiable Presentation in ID Token, Authentication Response (`vp_ldp`) section.
-
-
 ### Authorization Code Flow with Verifiable Presentation returned from the UserInfo endpoint
 
-Below are the examples when verifiable presentation is requested and returned from the UserInfo endpoint as part of OpenID Connect Authorization Code Flow. UserInfo response contains a `vp_jwt` or `vp_ldp` element with the Verifiable Presentation data. It can also contain `vc_jwt` or `vc_ldp` element with the Verifiable Credential data. 
+Below are the examples when verifiable presentation is requested and returned from the UserInfo endpoint as part of OpenID Connect Authorization Code Flow. UserInfo response contains a `verifiable_presentazions` element with the Verifiable Presentation data. It can also contain `verifiable_credentials` element with the Verifiable Credentials data. 
```

Comment:
```suggestion
Below are the examples when verifiable presentation is requested and returned from the UserInfo endpoint as part of OpenID Connect Authorization Code Flow. UserInfo response contains a `verifiable_presentations` element with the Verifiable Presentation data. It can also contain `verifiable_credentials` element with the Verifiable Credentials data. 
```